lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -182,6 +182,17 @@
     - '     - C(persistent): Changes are immediately and permanently written to the virtual disk. This is default.'
     - '     - C(independent_persistent): Same as persistent, but not affected by snapshots.'
     - '     - C(independent_nonpersistent): Changes to virtual disk are made to a redo log and discarded at power off, but not affected by snapshots.'
+    - ' - C(controller_type) (string): Type of disk controller. If this value is absent, disk will attach controller specified in C(hardware).'
```

Comment:
Can you also add an example in the module on the usage of this? Just to help people understand :)

Reply:
I add an example on the usage of this. Thank you for your comments.